libnetwork/iptables/firewalld.go

```diff
@@ -33,8 +35,13 @@ var (
 	connection *Conn
 
 	firewalldInitCalled bool
-	firewalldRunning    bool      // is Firewalld service running
-	onReloaded          []*func() // callbacks when Firewalld has been reloaded
+	firewalldRunning    bool // is Firewalld service running
+	// Time of the last firewalld reload, acquire firewalldReloadedAtMu before reading/writing.
+	firewalldReloadedAt   time.Time
+	firewalldReloadedAtMu sync.Mutex
```

Comment:
Perhaps we could use an `atomic.Value` for this;

```suggestion
	firewalldReloadedAt atomic.Value
```

Reply:
Done.